cfgov/v1/atomic_elements/organisms.py

```diff
@@ -141,6 +149,32 @@ class Meta:
         template = '_includes/organisms/table.html'
         label = ' '
 
+class AtomicTableInput(TableInput):
+    _id = 'table-block'
+    def render(self, name, value, attrs=None):
+        attrs.update({'id': self._id})
+        # Calling the grandparents render method and bypassing TableInputs,
+        # in order to control how we render the form.
```

Comment:
If I do that I would have to create an `init` method to assign the `table_options` value.